template/python3-flask-debian/index.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Alex Ellis 2017. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+from flask import Flask, request
+from function import handler
+#from gevent.wsgi import WSGIServer
+from gevent.pywsgi import WSGIServer
+
+app = Flask(__name__)
+
+@app.before_request
+def fix_transfer_encoding():
+    """
+    Sets the "wsgi.input_terminated" environment flag, thus enabling
+    Werkzeug to pass chunked requests as streams.  The gunicorn server
+    should set this, but it's not yet been implemented.
+    """
+
+    transfer_encoding = request.headers.get("Transfer-Encoding", None)
+    if transfer_encoding == u"chunked":
+        request.environ["wsgi.input_terminated"] = True
+
+@app.route("/", defaults={"path": ""}, methods=["POST", "GET"])
+@app.route("/<path:path>", methods=["POST", "GET"])
+def main_route(path):
+    ret = handler.handle(request.get_data())
+    return ret
+
+if __name__ == '__main__':
+    #app.run(host='0.0.0.0', port=5000, debug=False)
+
+    http_server = WSGIServer(('', 5000), app)
```

Comment:
Use 
```suggestion
    http_server = WSGIServer(('0.0.0.0', 5000), app)
```

binding to `0.0.0.0` resolves this error

```sh
Forking - python [index.py]
2020/04/04 14:15:13 Started logging stderr from function.
2020/04/04 14:15:13 Started logging stdout from function.
2020/04/04 14:15:13 OperationalMode: http
2020/04/04 14:15:13 Timeouts: read: 10s, write: 10s hard: 10s.
2020/04/04 14:15:13 Listening on port: 8080
2020/04/04 14:15:13 Writing lock-file to: /tmp/.lock
2020/04/04 14:15:13 Metrics listening on port: 8081
2020/04/04 14:15:13 stderr: Traceback (most recent call last):
2020/04/04 14:15:13 stderr:   File "index.py", line 33, in <module>
2020/04/04 14:15:13 stderr:     http_server.serve_forever()
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/baseserver.py", line 367, in serve_forever
2020/04/04 14:15:13 stderr:     self.start()
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/baseserver.py", line 305, in start
2020/04/04 14:15:13 stderr:     self.init_socket()
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/pywsgi.py", line 1490, in init_socket
2020/04/04 14:15:13 stderr:     StreamServer.init_socket(self)
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/server.py", line 146, in init_socket
2020/04/04 14:15:13 stderr:     self.socket = self.get_listener(self.address, self.backlog, self.family)
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/server.py", line 157, in get_listener
2020/04/04 14:15:13 stderr:     return _tcp_listener(address, backlog=backlog, reuse_addr=cls.reuse_addr, family=family)
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/server.py", line 252, in _tcp_listener
2020/04/04 14:15:13 stderr:     sock = GeventSocket(family=family)
2020/04/04 14:15:13 stderr:   File "/usr/local/lib/python3.7/site-packages/gevent/_socket3.py", line 114, in __init__
2020/04/04 14:15:13 stderr:     self._sock = self._gevent_sock_class(family, type, proto, fileno)
2020/04/04 14:15:13 stderr: OSError: [Errno 97] Address family not supported by protocol
2020/04/04 14:15:13 Forked function has terminated: exit status 1
```

I got this from https://github.com/locustio/locust/issues/556